SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -1153,8 +1153,9 @@ - (void)handleSendToMyself:(TSOutgoingMessage *)outgoingMessage
     [self handleMessageSentLocally:outgoingMessage];
 
     if (!(outgoingMessage.body || outgoingMessage.hasAttachments)) {
-        DDLogDebug(
-            @"%@ Refusing to make incoming copy of non-standard message sent to self:%@", self.logTag, outgoingMessage);
+        DDLogDebug(@"%@ Refusing to make incoming copy of non-standard message sent to self: %@",
```

Comment:
This looks like just a formatting change, but I don't remember when or why this would happen. If you know could you leave a comment?

Reply:
Done.